templates/.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,21 @@
+name: Auto merge
+
+on:
+  check_suite:
+    types:
+      - completed
+
+jobs:
+  auto-merge:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: cloudposse/actions/github/auto-merge@0.18.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          MERGE_LABELS: auto-merge,!wip
+          UPDATE_LABELS: auto-merge
+          MERGE_METHOD: "squash"
+          UPDATE_METHOD: "rebase"
+          MERGE_COMMIT_MESSAGE: "pull-request-description"
```

Comment:
Is `pull-request-description` a keyword that tells the action to use the description from the PR?

Reply:
Yep,
> MERGE_COMMIT_MESSAGE: The commit message to use when merging the pull request into the base branch. Possible values are automatic (use GitHub's default message), pull-request-title (use the pull request's title), pull-request-description (use the pull request's description), pull-request-title-and-description or a literal value with optional placeholders (for example Auto merge {pullRequest.number}). The default value is automatic.